tests/e2e/vLLM/test_vllm.py

```diff
@@ -201,7 +203,9 @@ def _run_vllm(self):
             # required by the kernel
             llm = LLM(model=self.save_dir, dtype=torch.float16)
         else:
-            llm = LLM(model=self.save_dir)
+            llm = LLM(
+                model=self.save_dir, gpu_memory_utilization=self.gpu_memory_utilization
```

Comment:
Wouldnt it be easier to just not pass in anything if we aren't setting it in our config? That way we dont have to worry about vllm changing their defaults

Reply:
Updated!